lib/rubocop/cop/lint/number_conversion.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop check usage of unsafe number conversions, eg. `foo.to_i`.
```

Comment:
I'd expand this into something explaining the purpose of the cop in a bit more detail.

Reply:
Added purpose of the cop along with complete examples.